docs/cody/clients/model-configuration.mdx

```diff
@@ -4,223 +4,14 @@
 
 <Callout type="note">For Sourcegraph Cloud customers, configuring the available LLM models requires contacting your Sourcegraph account team representative</Callout>
 
-Sourcegraph v5.6.0 or later supports the ability to choose between different LLM models, allowing developers to use the best model for Cody Chat as needed.
+Cody Enterprise can be configured using one of two methods:
+1. Legacy "Completions" Configuration
```

Comment:
I personally feel we shouldn't refer to this as the legacy option if that is our recommendation. Is there a specific reason why we're using this language?

Reply:
Reiterating myself:

> I would expect the docs phrasing to read more like ‘Versions of Sourcegraph after Aug 7th feature a fancy new way to configure models and model providers, this is early-access, it might work for you but if you don’t want to be an early tester than you should use completions instead <link>’